lib/install.js

```diff
@@ -9,17 +9,17 @@ const request = require('request');
 
 const _ = require('./helper');
 const logger = require('./logger');
-const Chromedriver = require('./macaca-chromedriver');
+//const Chromedriver = require('./macaca-chromedriver');
+const Versions = require('./versions');
 
-const CDNURL = process.env.CHROMEDRIVER_CDNURL || 'https://chromedriver.storage.googleapis.com';
-
-function getDownloadUrl() {
-  var url = util.format('%s/%s/chromedriver_%s.zip', CDNURL, Chromedriver.version, _.getExecFile(Chromedriver.version));
+function getDownloadUrl (version) {
+  var CDNURL = process.env.CHROMEDRIVER_CDNURL || 'https://chromedriver.storage.googleapis.com';
+  var url = util.format('%s/%s/chromedriver_%s.zip', CDNURL, version, _.getExecFile(version));
   logger.info('chromedriver cdn url: %s', url);
   return url;
 }
 
-function UnZipStream(distPath) {
+function UnZipStream (distPath) {
```

Comment:
https://github.com/xudafeng/slide/blob/master/archives/javascript-in-practice/src/style.md